src/module/system/settings/automation.ts

```diff
@@ -18,6 +18,7 @@ export class AutomationSettings extends SettingsMenuPF2e {
     protected static override get settings(): Record<ConfigPF2eListName, PartialSettingsData> {
         return {
             rulesBasedVision: {
+                prefix: "automation.",
```

Comment:
Can there be a default prefix?

Reply:
There can be, though given the number of places that have to prepend the prefix, as well as a helper, that seemed like the worse of two evils.

The other concern was how do we remove a prefix. Thinking now, `prefix: null` could work.

Reply:
If it's worse with a default, then that's fine.